cla_public/apps/contact/forms.py

```diff
@@ -136,6 +136,15 @@ class ContactForm(Honeypot, BabelTranslationsFormMixin, Form):
         validators=[
             IgnoreIf('callback_requested', FieldValue(NO))
         ])
+    email = StringField(
+        _(u'Email address'),
+        description=_(u'If you add your email we will send you the '
+                      u'reference number when you submit your details'),
```

Comment:
This is better formatted like this:
```python
description=_(
    u'If you add ...'
    u'reference ...'),
```
because the indentation level would be less arbitrary.